tests/FunctionalTest.php

```diff
@@ -204,20 +207,22 @@ private function createObjectManager(): ObjectManager
             $objectPersisterFactory,
             $objectIdentityMap,
             $classMetadataFactory,
-            $eventManager
+            $eventManager,
         );
 
         $objectFactory  = new ObjectFactory();
         $objectHydrator = new BasicObjectHydrator($objectManager);
-
-        $objectRepositoryFactory->addObjectRepository(User::class, new UserRepository(
+        /** @var ObjectRepositoryInterface<object> $userRepository */
+        $userRepository = new UserRepository(
```

Comment:
@greg0ire The excluded errors I mentioned in Skeleton-Mapper https://github.com/doctrine/skeleton-mapper/pull/39#issuecomment-1372879914 are affecting this line in this test. Any idea now how it can be fixed?